packages/@webex/plugin-cc/src/services/task/types.ts

```
@@ -2,511 +2,1095 @@
     type: string;
   };
 };
 
 /**
- * Response type for the task public methods
+ * Response type for task public methods
+ * Can be an {@link AgentContact} object containing updated task state,
+ * an Error in case of failure, or void for operations that don't return data
+ * @public
  */
 export type TaskResponse = AgentContact | Error | void;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)